pages/about.html

```diff
@@ -5,8 +5,40 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="styles/styles.css" rel="stylesheet">
 </head>
 <body>
+  <div class="container-about">
+      <header class="cell cell-1">
+          <nav>
+              <ul>
+                <li><a href="file:///Users/eunice/Developer/projects/personal-portfolio-site/pages/portfolio.html"> my projects</a></li>
+                <li><a href= "file:///Users/eunice/Developer/projects/personal-portfolio-site/pages/index.html">my page</a></li>
```

Comment:
Use relative links and **not** absolute ones.  These paths work on **your** computer, but on mine the absolute path is different.

```suggestion
        <li><a href="index.html"> Home</a></li>
        <li><a href= "portfolio.html"> my projects</a></li>

```